src/components/vmd-appointment-card.component.scss

```
@@ -66,3 +66,16 @@ vmd-appointment-metadata [slot='content'] a {
     text-decoration-width: 1px;
   }
 }
+
+a:hover {
```

Pour la navigation au clavier
```suggestion
a:hover, a:focus {
```